Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -144,15 +144,13 @@ To set up incident mirroring you need to:
 To use ServiceNow on Cortex XSOAR, ensure your service account has the following roles required to make API calls:  
 - Rest_api_explorer
 - Snc_platform_rest_api_access
-- Itil (Needed to access the sys_journal_field to allow comments, work notes, and files to be mirrored in and out. Incoming mirroring queries sys_journal_field for comments and work notes and therefore, access to sys_journal_table is also required.)  
-**Note:**  
-If your organization does not allow assigning the Itil role, you need to give service account elevated
-privileges to the sys_journal_field (see this ServiceNow community link for [giving elevated read access](https://community.servicenow.com/community?id=community_question&sys_id=b4051bf4db4c1cd823f4a345ca9619dc) and potential risks).
-- Read access to sys_journal_field is required for incoming mirroring.
+- itil or read/write access the specific tables you want to have access to. If you choose to give permissions only for specific tables, make sure you have the correct role so you have permissions to work with the relevant table. Keep in mind that these permissions may not suffice for managing records in some tables.
```

Comment:
Why did you change that?
This permission is basically for mirroring comments. What other use cases/tables can be?

I'm not sure we need to change this part

Reply:
@Shellyber  The new FR that was rolled out which uses 'Use Display Value' makes this as optional. Clients can now bypass the elevated privileges needed in order for comments and work notes to be accessed or mirrored. Hence this means that ITIL is no longer an essential privilege. They can either use ITIL or just ensure we have sufficient access to get all the records in the table. This is why I edited the notes to make it sequential and a little bit clearer to make sure clients understand their options.  Also noting that, if mirroring of comments and work notes fail, it would break the entire mirror cycle and nothing would be mirrored in. which makes this an important step to be clarified.

Reply:
@Bharathwaj-Murali I would probably write then that the Itil is not mandatory anymore since we have the new feature. 
But I don't think we need to change and add the rest.
I'll leave the third point as is and add a note that this is no longer required for mirroring comments and work notes and refer to the new functionality

Reply:
@Shellyber I did not change it. I just combined the footnote to the itil permission as they are related. And I moved the sys_journal_field to a separate point. I will reformat it.